app/src/main/java/com/example/parkingsystem/mvp/view/ParkingSpaceReservationView.java

```diff
@@ -63,4 +64,24 @@ public void showTimePickerDialog(TimePickerDialog.OnTimeSetListener listener) {
     public void enableButtonEnd() {
         binding.buttonParkingSpaceReservationPickerEnd.setEnabled(true);
     }
+
+    @Override
+    public int getParkingSpace() {
+        return Integer.parseInt(binding.inputParkingSpaceReservationPlace.getText().toString());
+    }
+
+    @Override
+    public int getSecurityCode() {
+        return Integer.parseInt(binding.inputParkingSpaceReservationCode.getText().toString());
+    }
+
+    @Override
+    public void showSaveDone() {
+        Context context = getContext();
+        Activity activity = getActivity();
+        if (context != null && activity != null) {
+            Toast.makeText(context, context.getString(R.string.toast_parking_space_reservation_save), Toast.LENGTH_SHORT).show();
+            activity.finish();
```

Comment:
please add a gif for this functionality, is the toast still visible after closing the activity?

Reply:
Added!